common/decklist.cpp

```diff
@@ -500,32 +547,36 @@ bool DeckList::loadFromStream_Plain(QTextStream &in)
             isSideboard = true;
         }
 
+        if (line.trimmed().isEmpty())
+            continue; // The line was "    " instead of "\n"
```

Comment:
Ok, so it's a sanity check on the line contents before starting to parse card name and qty.